pom.xml

```diff
@@ -49,7 +49,7 @@
       <tag>HEAD</tag>
   </scm>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>1.609.2</jenkins.version>
```

Comment:
BTW I don't see a clear reason for that core version discrepancy (introduced [here](https://github.com/andresrc/workflow-plugin/commit/0a5dbf7407c6fb60893954f89ef86d5a76fb9d47) without issue linked)